flow360/component/flow360_params/flow360_params.py

```diff
@@ -491,10 +492,17 @@ class Geometry(Flow360BaseModel):
     :class: Geometry component
     """
 
-    ref_area: Optional[AreaType.Positive] = pd.Field(alias="refArea", displayed="Reference area")
-    moment_center: Optional[LengthType.Point] = pd.Field(alias="momentCenter")
-    moment_length: Optional[LengthType.Moment] = pd.Field(alias="momentLength")
-    mesh_unit: Optional[LengthType] = pd.Field(alias="meshUnit")
+    ref_area: Optional[AreaType.Positive] = pd.Field(
+        alias="refArea", displayed="Reference area", default_factory=lambda: 1
+    )
+    moment_center: Optional[LengthType.Point] = pd.Field(
+        alias="momentCenter", default_factory=lambda: (0.0, 0.0, 0.0)
+    )
+    ##Note: moment_length does not allow negative components I failed to enforce that here after attempts
+    moment_length: Optional[LengthType.Moment] = pd.Field(
+        alias="momentLength", default_factory=lambda: (1.0, 1.0, 1.0)
+    )
+    mesh_unit: Optional[LengthType.Positive] = pd.Field(alias="meshUnit", default_factory=lambda: 1)
```

Comment:
lets remove `default_factory` completely and implement defaults in `to_solver()`.

Reply:
Sure